gems/sorbet-runtime/.rubocop.yml

```diff
@@ -84,6 +85,11 @@ Performance/CollectionLiteralInLoop:
   Exclude:
     # Ok to prioritize brevity over perf in tests
     - 'test/**/*'
+Performance/BindCall:
+  Exclude:
+    # Only runs when `bind_call` is not built into the stdlib
+    - 'lib/types/private/methods/call_validation_2_6.rb'
+    - 'lib/types/private/methods/call_validation.rb'
```

Comment:
I dunno, maybe we should just turn this off everywhere.

Reply:
I kind of prefer it this way to remind people to use bind_call elsewhere in the codebase